custom/icds_reports/utils.py

```diff
@@ -1414,6 +1476,9 @@ def get_data_for(filters, date):
             weighed=Sum('wer_weighed'),
             all=Sum('wer_eligible'),
         )
+        if show_test:
```

Comment:
I think this should be `not show_test` (there are some others that need to be switched as well)

Reply:
Fixed in all places